examples/http-proxy/http_proxy.py

```diff
@@ -3,29 +3,25 @@
 a simple http proxy example
 """
 import asyncio
-
-from datetime import datetime, timedelta, timezone
 import pathlib
 import shlex
 import sys
-
+from datetime import datetime, timedelta, timezone
```

Comment:
uh, how is this alphabetic order?

Reply:
`straight-style imports` before `from-style imports`, It is default configuration in many projects. I can change it to `force_sort_within_sections`